### DISCUSSION TITLE
Why does Mac Mouse Fix doesn't work when i move it to /Applications directory?

### DISCUSSION THREAD

**Discussion by avxkim** [noah-nuebling/mac-mouse-fix#582] [Questions]
It works only within ~/Downloads directory for me. Tried to uninstall, re-install, i can turn it on only within DOWNLOADS folder.

**Comment by yourGrand** [comment]
Same question

**Comment by noah-nuebling** [comment]
If you follow these steps, does that help?

1. Disable Mac Mouse Fix
2. Place it in the Applications folder
3. Restart your computer
4. Enable Mac Mouse Fix
